packages/react-network/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@shopify/react-network",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "A collection of components that allow you to set common HTTP headers from within your React application.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-network/README.md",
+  "dependencies": {
+    "@shopify/network": "^1.0.0",
+    "@shopify/react-effect": "^1.0.0",
+    "@types/koa": "^2.0.46"
```

Comment:
I think this should be a peer dependency so it can be used by consumers not using Koa (since you went to the trouble of generalizing the API in the first place).